content/ar/accessibility.html

```diff
@@ -0,0 +1,68 @@
++++
+title = "بيان إمكانية الوصول"
+meta_description = "Vote.gov is committed to making its digital content accessible to individuals with disabilities by meeting or exceeding the requirements of Section 508 of the Rehabilitation Act"
+translationKey = "accessibility"
++++
+<div class="usa-prose">
+<p>في vote.gov ، نحن ملتزمون بإمكانية الوصول. تتمثل سياستنا في ضمان حصول الجميع ، بما في ذلك الأشخاص ذوي الإعاقة ، على إمكانية الوصول الكامل والمتساوي إلى موارد التصويت. يعتمد موقع vote.gov على تقنيات معينة للتأكد من أنه يعمل بشكل جيد مع الأدوات التي تساعد الأشخاص ذوي الإعاقة في الوصول إلى المعلومات على مواقع الويب. وتشمل هذه التقنيات HTML و WAI-ARIA و CSS وجافا سكريبت. تأكد هذه التقنيات من أن الموقع سهل الاستخدام مع قارئات الشاشة ومكبرات الشاشة وبرامج التعرف على الكلام والتقنيات المساعدة الأخرى.</p>
+
+<p><strong>محتوى</strong></p>
+<ul>
+  <li><a href="#statement">بيان إمكانية الوصول</a></li>
+  <li><a href="#how-we-support-and-maintain">كيف ندعم إمكانية الوصول ونحافظ عليها</a></li>
+  <li><a href="#our-standards">معاييرنا</a></li>
+  <li><a href="#help-feedback-and-formal-complaints">المساعدة والتعليقات والشكاوى الرسمية</a></li>
+</ul>
+
+
+<h2 id="statement">بيان إمكانية الوصول</h2>
+<p>يلتزم فريق vote.gov بإنشاء موقع ويب يمكن الوصول إليه لجميع الناخبين ، بما في ذلك الأشخاص الذين يجدون صعوبة في الرؤية أو السمع أو تشغيل أجهزة الكمبيوتر أو الذين يواجهون تحديات معرفية أو تعليمية. نسعى أيضا للوصول إلى الأشخاص المحرومين ، بما في ذلك أولئك الذين لغتهم الأساسية ليست الإنجليزية. تعد إمكانية الوصول جهدا مستمرا ، ونسعى إلى تحسين موقعنا من خلال توفير تدريب منتظم للموظفين الذين يكتبون المحتوى الخاص بنا ، وينشئون مستنداتنا ، ويبنون موقعنا على الويب.</p>
+
+<h2 id="how-we-support-and-maintain">كيف ندعم إمكانية الوصول ونحافظ عليها</h2>
+<p><strong>نحن نضمن إمكانية الوصول إلى vote.gov من خلال:</strong></p>
+<ul>
+  <li>اختبار المحتوى يدويا لإمكانية الوصول إلى لوحة المفاتيح وقارئ الشاشة</li>
+  <li>استخدام عناوين الأقسام الدلالية لتنظيم المحتوى</li>
+  <li>السماح للأشخاص باستخدام لوحة المفاتيح للوصول إلى جميع الروابط والأجزاء التفاعلية من موقعنا على الويب</li>
+  <li>بما في ذلك وظيفة "التخطي إلى المحتوى الرئيسي"</li>
+  <li>إضافة رمز إلى الروابط الخارجية للإعلان عن "فتح في نافذة جديدة" لبرامج قراءة الشاشة</li>
+  <li>توفير نص بديل مفصل للصور والأيقونات والشعارات</li>
+  <li>إشراك أشخاص حقيقيين في ترجمة المحتوى لتقديم السياق وتجربة المستخدم</li>
+  <li>السماح للمستخدمين بتغيير حجم النص وفقا للتفضيل</li>
+</ul>
+
+<h2 id="our-standards">معايير إمكانية الوصول لدينا</h2>
+<p>نقوم بتصميم صفحات الويب الخاصة بنا لتلبية معايير <a href="https://www.section508.gov/">القسم 508</a> أو تجاوزها ، وهي المتطلبات الفنية التي تضمن امتثالنا لقانون القسم 508 الفيدرالي. نحن نلتزم أيضا بإرشادات W3C للوصول إلى محتوى الويب (WCAG) 2.1، والتي يمكن العثور عليها على موقع <a href="https://www.w3.org/tr/wcag21">W3C</a>. نحن نلبي معايير المستوى AA ، مما يعني أن المحتوى الخاص بنا متاح لمعظم الأشخاص في معظم الظروف.
```

Comment:
Please include ` {{< inenglish >}}` in the links that are not going to translated pages.